### PR TITLE
Updates LTI tool to standardize wording in prep for LTI as provider

### PR DESCRIPTION
**Pull request by lamagnifica**
In advance of course team documentation for the LTI-as-tool-provider feature, I tuned up the terminology in the existing LTI component topic and applied style guidelines.
@mhoeber @catong @srpearce

### CODE REVIEW

File: en_us/shared/exercises_tools/vitalsource.rst [hunk 1]
```diff
@@ -39,7 +39,9 @@ To create a VitalSource Bookshelf e-reader, you need the following information f
   about the **LTI Passports** policy key, see :ref:`LTI Information` in
   :ref:`LTI Component`.
 
-- The Vital Souce Book ID (VBID) for your e-book. This is a specific code that Vital Source creates for the book when Vital Source adds the e-book to its inventory.
+- The Vital Souce Book ID (VBID) for your e-book. This is a specific code that
```
Comment (catong) [inline en_us/shared/exercises_tools/vitalsource.rst]: missing an "r" in "Souce" -> "Source"

File: en_us/shared/exercises_tools/vitalsource.rst [hunk 2]
```diff
@@ -61,12 +63,12 @@ Source.
...
-
-   .. image:: ../../../shared/building_and_running_chapters/Images/LTIPolicyKey.png
-    :alt: Image of the Advanced Module List key in the Advanced Settings page, with the LTI value added
+#. Enter ``"lti"``. Make sure to include the quotation marks, but not the
+   period.The text in the **Advanced Module List** field should resemble the
+   following.
 
+   ``["lti"]``
+   
   .. note:: If the value field already contains text, place your cursor directly after the closing quotation mark for the final item, and then enter a comma followed by ``“lti”`` (make sure that you include the quotation marks). For example, the value for **Advanced Module List** may resemble the following:
```
Comment (catong) [inline en_us/shared/exercises_tools/vitalsource.rst]: There seem to be curly quotes in the 3rd line of this note.

Comment (catong) [inline en_us/shared/exercises_tools/vitalsource.rst]: "the following:" -> "the following example." or something similar to create a complete sentence and replace the colon with a period.

**Comment by catong** [comment]
@lamagnifica There are lots of great improvements here! :+1:

File: en_us/shared/exercises_tools/vitalsource.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/vitalsource.rst]: woo hoo! nice catch!

File: en_us/shared/exercises_tools/vitalsource.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/vitalsource.rst]: damn I thought I got all of the curly quotes!
thanks!

**Comment by lamagnifica** [comment]
It turns out that the VitalSource e-reader tool was removed from the documentation by DOC-928. Oddly, the file remains available and referenced by the index. Added a comment to the file to prevent future work on it.

**Pull request merged**